students/alex_skrn/lesson02/grid.py

```diff
@@ -0,0 +1,112 @@
+"""Print a 2x2 grid, define two print_grid functions."""
+# PART 1 - print a 2x2 grid, each cell 4 dashes large
+
+plus, minus, bar, space = '+', '-', '|', ' '
+numcells = 2
+numdashes = 4
+
+# construct the plus-and-dashes line, + - - - - + - - - - +
+plusline = ''
+for i in range(numcells):
+    plusline += plus + space + (minus+space)*numdashes
+plusline += plus
+
+# construct the bars-and-spaces line, |         |         |
+barline = ''
+for i in range(numcells):
+    barline += bar + space + space*2*numdashes
```

Comment:
You'll want to get spaces between your operators.  If you want to make it more readable, don't hesitate to throw some parens around even if order of operations doesn't need it:
```
bar_line += bar + space + (space * 2 * num_dashes)
```
This adds to the readability.